mmocr/models/textrecog/losses/ctc_loss.py

```diff
@@ -36,12 +38,15 @@ def __init__(self,
         self.ctc_loss = nn.CTCLoss(
             blank=blank, reduction=reduction, zero_infinity=zero_infinity)
 
-    def forward(self, outputs, targets_dict):
+    def forward(self, outputs, targets_dict, img_metas=None):
```

Comment:
When is the img_metas is None and when is it not None? How to control it? Can we set it in config?